lambda/remote_build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# On an Amazon Linux box, to initially set up.
+sudo yum install python36 python36-virtualenv
+sudo yum install sqlite-devel gcc libffi-devel openssl-devel
+sudo yum install git
+git clone https://github.com/18F/domain-scan.git
+git clone https://github.com/dhs-ncats/pshtt
+# May need to switch branches, depending.
```

Comment:
The above few lines are really here for documentation of how I set up the Amazon Linux instance post-AMI. The rest is repeatable (and to be run on the EC2 instance). This is obviously begging to become a Docker-based process to build these tasks, which is a TODO, but probably for a future PR.